### PR TITLE
chore: change yap version for support include tempalte

### PR DESCRIPTION
**Pull request by LiusCraft**
https://github.com/goplus/yap/pull/49 no merge, so use: `replace github.com/goplus/yap v0.7.2 => github.com/LiusCraft/yap v0.8.2`

### CODE REVIEW

File: cmd/gopcomm/gop_autogen.go [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+	"github.com/goplus/community/translation"
+	"github.com/goplus/yap"
+	_ "github.com/joho/godotenv/autoload"
+	"github.com/qiniu/x/xlog"
+	"golang.org/x/text/language"
+	"net/http"
 	"os"
 	"strconv"
 	"strings"
+<<<<<<< HEAD
```
Comment (IRONICBo) [inline cmd/gopcomm/gop_autogen.go]: Please update your merged code.

**Review by IRONICBo — CHANGES_REQUESTED** [review]

**Comment by IRONICBo** [comment]
@Kasey0727 Please review :)

**Review by Kasey0727 — APPROVED** [review]
LGTM

**Comment by LiusCraft** [comment]
@IRONICBo @Kasey0727 TBR

**Review by IRONICBo — APPROVED** [review]
LGTM

**Pull request merged**